src/main/java/mate/academy/rickandmorty/service/CharacterService.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.rickandmorty.service;
+
+import java.util.List;
+import mate.academy.rickandmorty.dto.internal.CharacterDto;
+
+public interface CharacterService {
+    CharacterDto getRandomCharacter();
+
+    List<CharacterDto> findCharactersByName(String name);
+
+    void setAllCharactersFromExternalDataBaseToInternalDataBase();
```

Comment:
Move this method to Client class
```suggestion
    void loadCharactersFromExternalApi();
```